cypress/support/pages/course_board/pageBoard.js

```diff
@@ -384,11 +388,11 @@ class Board {
 	}
 
 	enterTextToTextFieldInCard(textContent) {
-		cy.get('[data-testid="ckeditor"]').then((el) => {
+		cy.get('[data-testid="rich-text-edit-0-0"]').then((el) => {
```

Comment:
name this and the selector below it

Reply:
its like this before not from my PR, so I am not sure about the name yet.

Reply:
tho I updated it now.